newIDE/app/src/ObjectEditor/Editors/SpriteEditor/SpritesList.js

```diff
@@ -207,10 +266,16 @@ type Props = {|
   project: gdProject,
   resourcesLoader: typeof ResourcesLoader,
   resourceManagementProps: ResourceManagementProps,
+  editWith: (

```

Comment:
may I suggest calling this parameter `editDirectionWith` or `editDirectionWithExternalEditor`?
It feels otherwise a bit too generic as a name